site/en/reference/array_data_type.md

```diff
@@ -335,137 +275,330 @@ const fields = [
+    index_name: "vector_index",
+    params: { "nlist": 128 }
+});
+
+res = await client.describeIndex({
+    collection_name: "test_collection",
+    index_name: "vector_index"
+});
 
-After creating a collection, you can insert arrays such as the one demonstrated in [Overview](#overview).
+console.log("Vector index description: " + JSON.stringify(res));
+
+// Output:
+// Vector index description: {"index_descriptions":[{"params":[{"key":"params","value":"{\"nlist\":128}"},{"key":"index_type","value":"IVF_FLAT"},{"key":"metric_type","value":"COSINE"}],"index_name":"vector_index","indexID":"451543183233666062","field_name":"vector","indexed_rows":"0","total_rows":"0","state":"Finished","index_state_fail_reason":"","pending_index_rows":"0"}],"status":{"extra_info":{},"error_code":"Success","reason":"","code":0,"retriable":false,"detail":""}}
+```
 
 <div class="language-python">
 
-For more information on parameters, refer to [`insert()`](https://milvus.io/api-reference/pymilvus/v2.4.x/MilvusClient/Vector/insert.md) in the SDK reference.
+For more information on methods and parameters, refer to [prepare_index_params](https://milvus.io/api-reference/pymilvus/v2.4.x/MilvusClient/Management/prepare_index_params.md), [create_index](https://milvus.io/api-reference/pymilvus/v2.4.x/MilvusClient/Management/create_index.md), and [describe_index](https://milvus.io/api-reference/pymilvus/v2.4.x/MilvusClient/Management/describe_index.md).
 
 </div>
 
 <div class="language-java">
 
-For more information on parameters, refer to [`insert()`](https://milvus.io/api-reference/java/v2.4.x/v2/Vector/insert.md) in the SDK reference.
+For more information on methods and parameters, refer to [IndexParam](https://milvus.io/api-reference/java/v2.4.x/v2/Management/IndexParam.md) and [createIndex](https://milvus.io/api-reference/java/v2.4.x/v2/Management/createIndex.md).
 
 </div>
 
 <div class="language-javascript">
 
-For more information on parameters, refer to [`insert()`](https://milvus.io/api-reference/node/v2.4.x/Vector/insert.md) in the SDK reference.
+For more information on methods and parameters, refer to [createIndex](https://milvus.io/api-reference/node/v2.4.x/Management/createIndex.md), and [describeIndex](https://milvus.io/api-reference/node/v2.4.x/Management/describeIndex.md).
 
 </div>
 
+### Index array field
+
+Creating an index on a scalar field can improve the retrieval performance of queries on that field, which is optional but recommended for large datasets.
+
+In this example, we'll create an inverted index on the `color_coord` array field. This will allow us to speed up filtering based on this field. The inverted index demonstrates excellent overall performance, significantly outperforming raw data searches when data is not frequently accessed, and maintaining comparable performance even with frequent access and indexed columns. For more information on inverted indexes, refer to [Scalar Index](scalar_index.md#Inverted-indexing).
```

Comment:
`raw data seaches` may be replaced by `brute force filtering using raw data`, and
`access` -> `retrieval`
`accessed` -> `retrieved`.

And I think `even with frequent access and indexed columns` is a little confusing, maybe `even with frequent retrieval` is enough.